lemminx-maven/src/main/java/org/eclipse/lemminx/maven/searcher/RemoteRepositoryIndexSearcher.java

```diff
@@ -62,6 +62,8 @@
 	public static final RemoteRepository CENTRAL_REPO = new RemoteRepository.Builder("central", "default", "https://repo.maven.apache.org/maven2").build();
 	private final Set<RemoteRepository> knownRepositories;
 	
+	boolean disableCentralIndex = System.getProperty(getClass().getName() + ".disableCentralIndex") != null;
```

Comment:
@Mickaelistria this might be an ugly way to get a boolean from a system property, since it doesn't take into account the value passed (only null or not). Should I use a string and then convert it's value into a boolean?

Reply:
Ok will do.